docs/faq.md

```diff
@@ -143,6 +143,31 @@ and the architecture has some fundamental limitations that may be surprising.
 We encourage you to give it a try and report any issues you find to Go or
 connect-go to help bring WASM in Go forward.
 
+### Why am I seeing a "stream error: stream ID 5; INTERNAL_ERROR; received from peer" error message after X seconds? {#stream-error}
+
+It means that your [http.Server](https://pkg.go.dev/net/http#Server) has a
+`ReadTimeout` or `WriteTimeout` field configured. These fields apply
+to the entire operation duration, even for streaming calls. If an operation
+takes longer than the value specified, the server closes the stream and
+clients can see the above error message. The other timeout fields won't cause
+this error, and we
+ncourage you to set `ReadHeaderTimeout` in particular.
+
+### How do I close a client response stream in connect-go?
+
+On reading the response, a client calling `CloseResponse` will attempt to
+gracefully close the connection. `Close` does this by reading all of the
+remaining messages sent from the server until the final status message is
+received. If the status is in error, `Close` will return the wire error.
```

Comment:
We have a mix of calling it `CloseResponse` and `Close`. It's called `CloseResponse` for bidi streams and `Close` for server streams. Maybe we can clarify that and then just refer to it like "close" (without the fixed-width code font).
```suggestion
On reading the response, a client can call `CloseResponse` on bidirectional
streams or `Close` on server streams to gracefully close the connection.
This will discard any remaining messages sent from the server until the
final status message is received. If the status is an error, the close function
will return the wire error.
```